CONTRIBUTING.md

```diff
@@ -39,3 +39,10 @@ In general, we follow the ["fork-and-pull" Git workflow](https://github.com/susa
 6. Push changes to your fork.
 7. Open a PR in our repository and follow the PR template so that we can efficiently review the changes.
 
+## Coding standards
+
+We like to keep the code as consistent as possible, so please familiarize yourself with the existing style of the project, module of function you are updating and with the [PEP 8 - Style Guide for Python Code](https://peps.python.org/pep-0008/) if you're contributing to the Python code base or the [Microsoft&reg; C# Coding Conventions](https://learn.microsoft.com/en-us/dotnet/csharp/fundamentals/coding-style/coding-conventions) when contributing to the C# code base.
```

Comment:
> We like to keep the code as consistent as possible, so please familiarize yourself with the existing style of the project, module of function you are updating and with the [PEP 8 - Style Guide for Python Code](https://peps.python.org/pep-0008/) if you're contributing to the Python code base or the [Microsoft&reg; C# Coding Conventions](https://learn.microsoft.com/en-us/dotnet/csharp/fundamentals/coding-style/coding-conventions) when contributing to the C# code base.

I would reword as...

To keep the code as consistent as possible, please familiarize yourself with the existing style of the project. If you're contributing to the Python code base, follow the [PEP 8 - Style Guide for Python Code](https://peps.python.org/pep-0008/).  If contributing to the C# code base, follow the [Microsoft&reg; C# Coding Conventions](https://learn.microsoft.com/en-us/dotnet/csharp/fundamentals/coding-style/coding-conventions).